content/docs/reference/routes/tls-upstream-allow-renegotiation.md

```diff
@@ -0,0 +1,21 @@
+---
+id: tls-upstream-allow-renegotiation
+title: TLS Upstream Allow Renegotiation
+keywords:
+- reference
+- TLS Upstream Allow Renegotiation
+---
```

Comment:
Ideally we should include a "description:" here as well; it can just reiterate the text on line 15, so
---
description: TLS Upstream Allow Renegotiation controls whether server-initiated TLS renegotiation is allowed for upstream servers.
---